test_src/sljitTest.c

```diff
@@ -345,11 +345,14 @@ static void test4(void)
 	buf[3] = 0;
 
 	sljit_emit_enter(compiler, 0, SLJIT_ARGS2(W, P, W), 3, 2, 0, 0, 0);
+    /* buf[0] */
+    sljit_emit_op2(compiler, SLJIT_SUB, SLJIT_MEM0(), (sljit_sw)&buf[0], SLJIT_IMM, 0, SLJIT_MEM0(), (sljit_sw)&buf[1]);
+    /* buf[2] */
```

Comment:
Something is wrong with the indentation. Is it space instead of tabs?

Reply:
It was a VS Code indentation problem. It should be fixed now.